devdox/app/services/git_tokens_service.py

```
@@ -5,7 +5,7 @@
 from app.repositories.git_label_repository import TortoiseGitLabelStore
 from app.schemas.basic import RequiredPaginationParams
 from app.schemas.git_label import GitLabelResponse
-from app.utils.auth import AuthenticatedUserDTO
+from app.utils.auth import AuthenticatedUserDTO, UserClaims
```

Ruff (0.11.9)

`app.utils.auth.AuthenticatedUserDTO` imported but unused

Remove unused import: `app.utils.auth.AuthenticatedUserDTO`

(F401)